lib/topological_inventory/schema/default.rb

```diff
@@ -51,12 +54,27 @@ def add_secondary_refs_name(builder)
         builder.add_properties(:secondary_refs => {:by_name => [:name]})
       end
 
+      def add_secondary_refs_uid_ems(builder)
+        builder.add_properties(:secondary_refs => {:by_uid_ems => [:uid_ems]})
+      end
+
       def add_containers
         add_collection(:containers) do |builder|
           add_default_properties(builder, manager_ref: [:container_group, :name])
           builder.add_default_values(:tenant_id => ->(persister) { persister.manager.tenant_id })
         end
       end
+
+      def add_cross_link_vm
+        add_collection(:cross_link_vm) do |builder|
+          builder.add_properties(
+            :arel           => Vm,
```

Comment:
```Vm.where(:tenant_id => manager.tenant_id)```

I think manager is accessible here, right?

Reply:
Oh yeah good catch